frontend/src/components/pages/admin/UsersPage.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
```

Comment:
No major changes were actually made to this component, I just renamed it from `AdminPage` to `UsersPage` and removed the lines that are now included in `AdminDashboard` (which wraps this component)